modules/bbp-simulation/src/main/resources/schemas/hbphackaton/simulation/spatialindex/v0.1.0.json

```diff
@@ -0,0 +1,26 @@
+{
+  "@context": [
+    "{{base}}/contexts/nexus/core/schema/v0.1.0",
```

Comment:
Same as above: The nsg schema context could be used to reduce context to one line